stable/spark/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: spark
-version: 0.1.6
+version: 0.1.7
```

Comment:
since spark is upgrade to version 2.2.0, do you think the minor version of the chart needs to be bumped?
Typically we bump the chart minor version if an existing deployment cannot be upgrade to the new release. This could be the case if there are backwards incompatible in the application image or the chart itself.

Reply:
I don't know, the chart itself has not encountered a "major" change, it's just a version. The deployments, number of replicas, etc... are the same. But if you think it should be `0.2.0` or even 1.0.0, it's no problem for me

Reply:
But yes you are right, spark 2 has breaking changes with spark 1

Reply:
then i'd suggest bumping the version to `0.2.0`

Reply:
@sameersbn done